.github/workflows/pr_test_build.yml

```diff
@@ -148,6 +148,7 @@ jobs:
           echo "const moralisApiKey = '${{ secrets.MORALIS_API_KEY }}';" >> lib/.secrets.g.dart
           echo "const polygonScanApiKey = '${{ secrets.POLYGON_SCAN_API_KEY }}';" >> cw_evm/lib/.secrets.g.dart
           echo "const ankrApiKey = '${{ secrets.ANKR_API_KEY }}';" >> cw_solana/lib/.secrets.g.dart
+          echo "const nano2ApiKey = '${{ secrets.NANO2_API_KEY }}';" >> cw_nano/lib/.secrets.g.dart
```

Comment:
what's that? and what is the value for it? and how it's not added to github actions and Ramo tested it correctly? is it not mandatory? or will just not affect the flow of nanogpt

Reply:
I sent a message about this a while back, but essentially Esteban requested that we use an api key

Someone was abusing node services and using an api key is a more secure way of authenticating than the current cake wallet header we send on requests

you can make an account / get an api key at rpc.nano.to

I did forget to mention I made an api key and had Esteban hook it up on the backend with 10 nano worth of credits just so we would stop getting user complaints for the time being while we figure out what we want to do (should last a few weeks at least)

once you make an account / have an api key just let me / Esteban know and he can switch over the account / credits